roles/install/tasks/10_manifest.yml

```diff
@@ -1,10 +1,12 @@
 - name: Manifest | Pull down manifest
   get_url:
-    url: https://subscription.rhsm.redhat.com/subscription/consumers/{{ manifest_uuid }}/export
+    url: >-
+      https://subscription.rhsm.redhat.com/subscription/consumers/
+      {{ manifest_uuid }}/export
```

Comment:
This is going to give you a space in the URL. You either should just disable the line-length rule for this line by adding `#yamllint disable-line rule:line-length` on the line above the URL or you need to put the line break inside of the Jinja template string like this:

```yaml
url: >-
  https://subscription.rhsm.redhat.com/subscription/consumers/{{
  manifest_uuid }}/export
```

Disabling the rule might be better, here, because the line break in there might look ugly to people.